### PR TITLE
Initializing vagrant and flask app

### PR DESCRIPTION
**Pull request by mosesnagar**
Creating a vagrantfile and script to install python and flask.
Initialize flask app.
Adding main route to present "hello world".

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/bionic64"
+  config.vm.provision :shell, path: "bootstrap.sh"
+  config.vm.network :forwarded_port, guest: 5000, host: 1111
+  config.vm.provider "virtualbox" do |v|
```
Comment (lioramilbaum) [inline Vagrantfile:5]: Is there a specific reason to use a host port which is not 5000?

File: tutor/routing.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from flask import render_template
```
Comment (lioramilbaum) [inline tutor/routing.py:1]: It is not used in this file

File: Vagrantfile (hunk 1 above)
Comment (mosesnagar) [reply Vagrantfile:5]: You're right, there is no reason for that.

File: tutor/routing.py (hunk 2 above)
Comment (lioramilbaum) [reply tutor/routing.py]: Do add a comment or/and emoji (thumb up) when you accept a review comment.

**Pull request merged**